akka-stream/src/main/scala/akka/stream/impl/FanoutProcessor.scala

```diff
@@ -79,11 +79,9 @@ private[akka] abstract class FanoutOutputs(val maxBufferSize: Int, val initialBu
     case SubscribePending ⇒
       subscribePending()
     case RequestMore(subscription, elements) ⇒
-      // FIXME can we avoid this cast?
       moreRequested(subscription.asInstanceOf[ActorSubscriptionWithCursor[Any]], elements)

```

Comment:
strange, I thought I tried that, but it works, changing to that instead:

    case RequestMore(subscription: ActorSubscriptionWithCursor[Any], elements) ⇒
       moreRequested(subscription, elements)